app/assets/stylesheets/components/_navbar.scss

```diff
@@ -36,3 +36,21 @@
     font-size: color;
   }
 }
+
+@media (min-width: 100px) and (max-width: 575px) {
+  .navbar-collapse {
+    background: white;
+    border: solid 1px lightgrey;
+    border-radius: 12px;
+    padding: 0.5em;
+    align-items: flex-end;
```

Comment:
```suggestion
    margin-left: auto;
    flex-basis: auto;
    // align-items: flex-end;
```

you can try this to make the navbar not full width :D

<img width="354" alt="Screenshot 2023-11-28 at 11 50 22" src="https://github.com/wyshlist/wyshlist/assets/74747669/54ae8699-612e-48d9-9947-8fa49bae65b3">

Reply:
I was a little bit struggling with this, thanks!

Reply:
Well, at the end even with the help of Daniel we couldn't solve this. With your code it gives this !
<img width="1680" alt="Capture d’écran 2023-11-28 à 15 50 37" src="https://github.com/wyshlist/wyshlist/assets/122105569/9b599e49-e8aa-4972-84d2-10dbfa00538d">

Reply:
Oh nooo :( Let's go with the first one first then